src/pdsfile/pds3file/rules/COVIMS_0xxx.py

```
@@ -2,12 +2,14 @@
 # pds3file/rules/COVIMS_0xxx.py
 ##########################################################################################
 
-import pdsfile.pds3file as pds3file
-from pdsfile.pdsfile import abspath_for_logical_path
 import os
-import translator
 import re
 
+import translator
+
+import pdsfile.pds3file as pds3file
```

Ruff (0.16.1)

[warning] Use `from pdsfile import pds3file` in lieu of alias

Replace with `from pdsfile import pds3file`

(PLR0402)